contracts/DCAHub/DCAHubConfigHandler.sol

```diff
@@ -29,6 +29,8 @@ abstract contract DCAHubConfigHandler is DCAHubParameters, AccessControl, Pausab
   /// @inheritdoc IDCAHubConfigHandler
   uint16 public platformFeeRatio = 2500; // 25%
 
+  mapping(address => bool) internal _allowedTokens;
```

Comment:
Why internal? It woud be nice to be able to query this from etherscan for example